sync/__init__.py

```diff
@@ -1,4 +1,4 @@
 """Library for leveraging the power of Sync"""
-__version__ = "0.5.3"
```

Comment:
Because this is part of a major new initiative (i.e. instance size optimization), does this deserve a more significant version number update. Should we make it 0.6.0? More generally, how should we be thinking about upgrades to version  number and what constitutes a major vs. a minor one, etc.?

Reply:
Hmmm good question, I'm not sure what the versioning scheme has been in the past but agree that this is more than a simple fix and because it's not breaking changes the minor version update here would make sense. Unless @gorskysd or @singhals has some input here, I'd say we move this up to `0.6.0`

Reply:
@syncbrian Based on our conversation from slack, I've changed the version back down to 0.5.3 (i.e. left it unchanged) with the thinking that the version number will be updated to 0.6.0 at the next release. Let me know if this sounds right to you

Reply:
yup that sounds right!